content/root/best-practice-guide.textile

```diff
@@ -0,0 +1,146 @@
+---
+title: Best Practice Guide
+section: root
+index: 30
+jump_to:
+  Contents:
+    - Ably 101
+    - Using Ably like a pro
+---
+
+h2(#title). Ably Best Practice Guide
+
+This best practice guide aims to quickly get you to understand the best way to make use of Ably's Realtime platform for your applications. These recommendations include some important considerations that are otherwise prone to easy ignoration while skimming through the docs. 
```

Comment:
Replace "quickly get you to" with "quickly help you to".

ignoration is an odd word IMHO, I believe it generally means the state of being ignorant, which is not what we mean and could be considered offensive.

Perhaps: "These recommendations include some important considerations that may otherwise be missed when reading through our documentation"